src/Mealz/MealBundle/Resources/views/Meal/partials/meal.html.twig

```diff
@@ -14,7 +14,7 @@
     {% endif %}
 {% endif %}
 
-<div class="meal-row{{ dish.isCombinedDish ? 'combined-meal' }} "
+<div class="meal-row {{ dish.isCombinedDish ? 'combined-meal' }} "
```

Comment:
I would put the space before 'combined-meal' in order to render the space only if we have that additional class. Otherwise the HTML will look like class="meal-row ".